commune_app/all_models/communes.py

```diff
@@ -0,0 +1,25 @@
+from django.db import models
+from .users import User
+from .chores import Chore
+from .votes import Vote
+
+
+class Commune(models.Model):
+    """
+      add something
+    """
+    id = models.IntegerField(unique=True, primary_key=True, auto_created=True)
+    users = models.ManyToManyField(User)
+    votes = models.ForeignKey(Vote, on_delete=models.CASCADE)
+    chores = models.ForeignKey(Chore, on_delete=models.CASCADE)
+
+    wallet = models.IntegerField()
+    name = models.TextField()
```

Comment:
maybe it is also better to add unique=True like:
```suggestion
    name = models.TextField(unique=True)
```